disperser/dataapi/server_v2.go

```diff
@@ -105,13 +105,13 @@ func (s *ServerV2) Start() error {
 	{
 		blob := v2.Group("/blob")
 		{
-			blob.GET("/blob/feed", s.FetchBlobFeedHandler)
-			blob.GET("/blob/:blob_key", s.FetchBlobHandler)
+			blob.GET("/blobs/feed", s.FetchBlobFeedHandler)
```

Comment:
nit: Same comment: `FetchBlob` -> `FetchBlobs`?

Reply:
FetchBlob is fetching exactly one blob. The edge case is "FetchBlobsFeed" or "FetchBlobFeed", which I am not sure which one, maybe "blob" is adjective here so "BlobFeed" sounds right?